hbase-it/pom.xml

```diff
@@ -336,6 +338,11 @@
       <activation>
         <jdk>[1.11,)</jdk>
       </activation>
+      <properties>
+        <failsafe.jdk8.flags/>
```

Comment:
Nice trick, unsetting the property in the profile.

If you can override with profiles, why not have a `<failsafe.profile.overrides>`? It can default to `${failsafe.jdk8.flags}` and in the JDK11+ profile, it can be redefined with `${hbase-surefire.jdk11.flags}`. Then the value of `<argLine>` need only be specified once. Would that work?

Reply:
Thanks @ndimiduk. It's good suggestion. Will change accordingly.

Reply:
Handled it in new commit.